dolphin/api/extensions.py

```diff
@@ -100,15 +98,15 @@ def show(self, req, id):
             # NOTE(dprince): the extensions alias is used as the 'id' for show
             ext = self.extension_manager.extensions[id]
         except KeyError:
-            raise webob.exc.HTTPNotFound()
+            raise exception.NotFound()
 
         return dict(extension=self._translate(ext))
 
     def delete(self, req, id):
-        raise webob.exc.HTTPNotFound()
+        raise exception.NotFound()
```

Comment:
Please check NotFound used directly

Reply:
the logic doesn't go into the specific business logic, so here it's NotFound.

Reply:
Ok, more than logic it’s coding flow

Reply:
thank you, the purpose of pr is not change the exception. We should use and define exceptions wisely